src/main/java/mate/academy/model/Order.java

```diff
@@ -18,6 +20,9 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @OneToMany
+    @JoinTable(name = "orders_tickets",
+            joinColumns = @JoinColumn(name = "order_id"),
+            inverseJoinColumns = @JoinColumn(name = "ticket_id"))
     private List<Ticket> tickets;
     private LocalDateTime orderDate;
```

Comment:
what about this column?

Reply:
> also let's test 2 orders with 2 tickets in each order (added comment above)

I have been modifying the cinema. The query will return n orders with n tickets if you have n orders with n tickets in each order?